README.md

```diff
@@ -10,35 +10,87 @@ Common components for Ronas IT projects.
 
 At the moment this library contains the following components:
 
-## `AppPressable`
+## UI-components
+
+### 1. `AppPressable`
 
 Component can be used in the same way as the built-in `Pressable` component, but it also includes opacity control.
 
-### Props
+#### Props
 
 - `pressedOpacity`: Opacity value. Default is 0.4.
 
-### Usage
+#### Usage
 
 ```jsx
 <AppPressable style={styles.button} pressedOpacity={0.5}>
   <Text>Press Me</Text>
 </AppPressable>
 ```
 
-## `AppSafeAreaView`
+### 2. `AppSafeAreaView`
 
 Is a component for granular control of safe area edges on each screen. The difference from `SafeAreaView` in [react-native-safe-area-context](https://www.npmjs.com/package/react-native-safe-area-context) is that the container adds padding to the elements inside it, rather than to the entire screen, making it more flexible for use.
 
-### Props
+#### Props
 
 - `edges`: An array indicating which edges of the screen to respect. Possible values are 'top', 'right', 'bottom', 'left'. Defaults to all edges.
 - `style`: Custom styles to apply to the view. Note that padding values will be adjusted to respect safe area insets.
 
-### Usage
+#### Usage
 
 ```jsx
 <AppSafeAreaView edges={['top', 'bottom']} style={styles.container}>
   <Text>Content goes here</Text>
 </AppSafeAreaView>
-````
+```
+
+## Services
+
+### 1. `PushNotificationsService` and `usePushNotifications` hook
```

Comment:
I'm not sure that we need a code example here because it will take up a lot of space. And this is a simple hook with a large number of arguments.